spec/features/buildings_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Building Creation' do
+  it 'succeeds' do
+    visit 'buildings/new'
+    fill_in 'building_name', with: 'Silliman'
+    expect { click_on 'Create' }.to change { Building.count }.from(0).to(1)
```

Comment:
One thing that's been recommended to me with acceptance tests is to also make assertions against the UI, not against the backend. For example, here we might check that a success flash is displayed and that the resulting page has an element indicating that the operation worked (e.g. a title if we go to the show view or a link if we go to the index view, etc). This keeps the acceptance tests as completely representative of the user experience, and you rely on your controller / views to do the right thing (and test them in isolation if necessary).

Thoughts?